docs/chains/chaining-chainlets.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Chaining Chainlets
+description: "How two *chainlets* can be combined in a *chain*"
+---
+<Warning>This is a beta feature and subject to breaking changes.</Warning>
+
+# Table of Contents
+
+* [Introduction](/chains/intro)
+* [Getting Started](/chains/getting-started)
+* [Chaining Chainlets](/chains/chaining-chainlets)
+* [Dependencies & Resources](/chains/deps-and-resources)
+* [Reference](/chains/full-reference)
+
+
+In the [previous section](chains/getting-started), we created a simple chain with a single chainlet.
+In this section, we'll add a second chainlet to the chain. The toy example builds on top of
+the previous Hello World example, and prints out "Hello World" with a random number of exclamation points at the end.
+
+The computational graph for this chain looks like this:
+
+<img src="/images/second-chainlet.png" />
+
+{/* <Note> */}
+{/* In practice, for a chain this simple, you wouldn't need to use two chainlets. However, this example is useful for demonstrating how to use multiple chainlets in a chain. */}
```

Comment:
I think this comment could be dropped. Most tutorials of any software use dummy/demo things that aren't super realistic - that's a common/reasonable expectation, that we don't need to explain.